app/build.gradle.kts

```diff
@@ -128,6 +128,9 @@ dependencies {
 
     implementation(libs.zoomable)
     implementation(libs.zoomable.image.coil)
+
+    // Rich Edit Text
```

Comment:
Can you please write this dependency like others are written please check them out ad write the dependency in version catalog format.

Reply:
sure, it's added now.